setup.cfg

```diff
@@ -5,3 +5,8 @@ universal = 1
 exclude = urls.py,tasks.py,*migrations*,*bower_components*
 max-line-length = 100
 ignore = W503
+
+[flake8]
+exclude = .git,__pycache__,urls.py,tasks.py,*migrations*,*bower_components*,*docs*,__*__,
+max-line-length = 100
+ignore = F401,W503
```

Comment:
add a new line at the end of your files

Reply:
@tibetegya, there is a new line but I don't know why it is not being shown here.